language/stdlib/modules/LibraBlock.move

```diff
@@ -4,24 +4,25 @@ module LibraBlock {
     use 0x1::CoreAddresses;
     use 0x1::Event;
     use 0x1::LibraSystem;
+    use 0x1::LibraConfig;
     use 0x1::LibraTimestamp;
     use 0x1::Signer;
 
     resource struct BlockMetadata {
-      // Height of the current block
-      // TODO: should we keep the height?
-      height: u64,
-      // Handle where events with the time of new blocks are emitted
-      new_block_events: Event::EventHandle<Self::NewBlockEvent>,
+        // Height of the current block
+        // TODO: should we keep the height?
```

Comment:
Just checking: are you adding one or two spaces? (Can't see from this view). Please ensure indentation is 4.

Reply:
Yes I have changed all the indentation to be four spaces.